modules/battery_manager/README.md

```diff
@@ -0,0 +1,58 @@
+# Battery Manager
+
+## Purpose
+The Battery Manager module allows users to **browse**, **add**, **edit**, **activate** and **deactivate** entries 
+in the Test Battery. The Test Battery is used to determine which Instruments are administered at different timepoints.
+
+## Intended Users
+The Battery Manager module is used by study administrators.
+
+## Scope
+The Battery Manager module provides a tool for browsing, adding, editing, activating, and deactivating entries in the
+the Test Battery.
+
+### Browse
+The **browse** functionality is under the `Browse` tab. Users can browse entries in the Test Battery and filter the entries using the `Selection Filters`.
+
+### Add
+The **add** functionality is under the `Add` tab. Users can add an entry by filling out the required fields. If the entry already exists in the Test Battery table, the entry cannot be added.
+
+### Edit
+The **edit** functionality can be accessed in the Menu Table under the `Browse` tab. Users can navigate to an entry's edit window by clicking on the corresponding `Edit` link in the `Edit Metadata` column. The edit window will have a form that is populated with the values of that entry. Editing entails deactivating the original entry and creating a new entry with the edited values.
+The following rules apply when editing:
+- Users cannot edit an entry if they have made no changes in the form.
+- Users cannot edit an entry if it becomes the same as another active entry in the Test Battery.
+- If the edited entry becomes the same as another deactivated entry in the Test Battery,
+  users will be given the option to activate the other entry and deactivate the original one.
+
+### Activate
+The **activate** functionality can be accessed in the `Browse` tab, `Edit` window, and the `Add` tab.
+
+#### Browse tab
+In the `Change Status` column of the Menu Table, users can directly activate an entry by pressing the **Activate** button.
+
+#### Interactions with LORIS
+*For Rida: Specify that changing the values in this table dictates what goes where in timepoints*
```

Comment:
?

Reply:
I didn't really touch this entire readme, so maybe this was leftover from the previous PR? @ridz1208 could you comment on this at all?